Demo/Application/Base/Settings/Settings.bundle/Root.plist

```diff
@@ -103,7 +103,6 @@
 			<array>
 				<integer>0</integer>
 				<integer>1</integer>
-				<integer>2</integer>
```

Comment:
This was left over from when we had a 3rd auth type, PayPalIDToken. There should only be 2 entries here (for client token & tokenization key)